linting/stylelint-config-ffe/index.js

```diff
@@ -18,7 +18,7 @@ module.exports = {
         ],
         'no-missing-end-of-source-newline': null,
         'number-leading-zero': null,
-        'max-nesting-depth': 3,
+        'max-nesting-depth': 5,
```

Comment:
With `ignoreAtRules: ['media']` a `max-nesting-depth` of 4 should be enough. I.e.:

```javascript
'max-nesting-depth': [4, { ignoreAtRules: ['media'] }],
```

Reply:
Fixed in 03e9c4c